intercom_flutter/android/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.1'
+        classpath 'com.android.tools.build:gradle:4.2.2'
```

Comment:
Latest is 7.0.4

Reply:
should we update to 7.0.4? I think the Gradle plugin needs to be updated to the latest as well.